### PR TITLE
Time map batch

### PR DESCRIPTION
**Pull request by EivydasKoc**
Add quick time-map-batch method

### CODE REVIEW

**Comment by danielnaumau** [comment]
You also need to deal with RPM limits. Can you add a function which will limit SDK calls per minute?

**Review by danielnaumau — DISMISSED** [review]
Plus can you also add an example how to use this new time map?

**Comment by EivydasKoc** [comment]
> You also need to deal with RPM limits. Can you add a function which will limit SDK calls per minute?

Users may opt-in by passing int a RateLimiter instance to the client which handles all rate limiting for all requests, including this one.

**Comment by danielnaumau** [comment]
> > You also need to deal with RPM limits. Can you add a function which will limit SDK calls per minute?
> 
> Users may opt-in by passing int a RateLimiter instance to the client which handles all rate limiting for all requests, including this one.

Could you add an example how to use this new time map with a rate limiter?

File: src/client/client.ts [hunk 1]
```diff
@@ -164,6 +165,44 @@ export class TravelTimeClient {
...
+  async timeMapBatch<T extends keyof TimeMapResponseType>(
+    bodies: TimeMapRequest[],
+    format: T,
+    chunkSize?: number,
+  ): Promise<BatchedResponse<TimeMapResponseType[T]>> {
+    const responses: TimeMapResponseType[T][] = [];
+    const errors: Array<{ index: number; error: Error }> = [];
+
+    for (let i = 0; i < bodies.length; i += chunkSize || 10) {
+      const chunk = bodies.slice(i, i + (chunkSize || 10));
```
Comment (danielnaumau) [inline src/client/client.ts:186]: how would you slice intersections and unions here?

Comment (EivydasKoc) [reply src/client/client.ts:186]: discussed on call, no changes needed

**Pull request merged**